.github/workflows/packaging.yml

```diff
@@ -10,8 +10,17 @@ on:
       - '*'
 
 jobs:
+  version-check:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Check module version
+        uses: tarantool/actions/check-module-version@igrishnov/gh-204-add-hardcoded-versioning-support
+        with:
+          module-name: 'expirationd'
+
```

Comment:
Can this code be extracted into a separate file? It is now have been repeated in `packaging.yml` and `publish.yml`.

Reply:
To minimize duplication, we have separated this into reusable action `tarantool/actions/check-module-version`. To be honest, I don't see how to do it even better yet

Reply:
Can we move this step into a separate workflow? Or is it not possible to be depended to a step in other workflows?

Reply:
> Or is it not possible to be depended to a step in other workflows?

Unfortunately, yes